hack/generated/pkg/genruntime/helpers.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package genruntime
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/apiutil"
+)
+
+func NewObjectFromObject(obj client.Object, scheme *runtime.Scheme) (client.Object, error) {
```

Comment:
Need comment

Reply:
Is there a better name for this method? Maybe `NewInstanceFromExemplar()`?

Reply:
I think `NewObject` is important because the thing we're making is a `client.Object`. The second half though I don't have an issue changing: `NewObjectFromExemplar`?

I've made that change and also added a comment.

Reply:
Looks good.